azext_iot/operations/hub.py

```diff
@@ -735,7 +739,7 @@ def iot_device_module_method(cmd, device_id, module_id, method_name, hub_name=No
 
 # Utility
 
-def iot_get_sas_token(cmd, hub_name=None, device_id=None, policy_name='iothubowner',
+def iot_get_sas_token(cmd, hub_name=None, device_id=None, policy_name='iothubowner', module_id=None,
```

Comment:
I would like to see consistent order of params.  For example, below you have module_id right after device_id, and here it is after policy_name.

Reply:
fixed